src/WorkflowExecuter/Services/WorkflowExecuterService.cs

```diff
@@ -596,7 +596,7 @@ private async Task<WorkflowInstance> CreateWorkflowInstanceAsync(WorkflowRequest
                 OutputDirectory = $"{payloadId}/workflows/{workflowInstanceId}/{executionId}",
                 ResultMetadata = { },
                 InputParameters = newInputParameters,
-                PreviousTaskId = previousTaskId
+                PreviousTaskId = previousTaskId is not null ? previousTaskId : string.Empty
```

Comment:
Could this be simplified to...

`previousTaskId ?? string.Empty`

?

Reply:
Fixed